src/components/node-list/index.js

```diff
@@ -207,6 +253,52 @@ const NodeListProvider = ({
       onToggleNodeSelected(null);
     }
   };
+
+  // Updates the URL parameters when the filter is reset.
+  const updateUrlParamsOnResetFilter = () => {
```

Comment:
I think this logic can go in use-generate-pathname.js

Reply:
done.